src/modules/QtQuick/Animation.js

```diff
@@ -1,19 +1,21 @@
-QmlWeb.registerQmlType({
-  module: "QtQuick",
-  name: "Animation",
-  versions: /.*/,
-  baseClass: "QtQml.QtObject",
-  enums: {
+QmlWeb.registerQmlType(class {
+  static module = "QtQuick";
+  static element = "Animation";
```

Comment:
`name` is already present for all classes (and functions), it's a string there, and in this specific case it's an empty string because the class is unnamed.

```shell
> (function x(){}).name
'x'
> (class {}).name
''
> (class Foo {}).name
'Foo'
```

Yes, `element` isn't perfect, but I wasn't able to come up with something better. Any suggestions?